src/SprykerEco/Client/Payone/ClientApi/Request/AbstractRequest.php

```diff
@@ -134,11 +134,11 @@ public function getMid(): ?string
     }
 
     /**
-     * @param string $aid
+     * @param string|null $aid
```

Comment:
Here too ...

Reply:
It's because we have such cases, when we put something which has string|null as a return type hint to this variable.